script.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+civicrm_initialize();
+
+$config = CRM_Core_Config::singleton(); ##
+global $databases;
+$drupal_prefix = '';
+if (isset($databases['default']['default']['prefix'])) {
+    if (is_array($databases['default']['default']['prefix'])) {
+        $drupal_prefix = $databases['default']['default']['prefix']['default'];
+    } else {
+        $drupal_prefix = $databases['default']['default']['prefix'];
+    }
+}
+
+if ($config->dsn != $config->userFrameworkDSN || !empty($drupal_prefix)) {
+    $dsnArray = DB::parseDSN($config->dsn);
+    $tableNames = CRM_Core_DAO::getTableNames();
+    asort($tableNames);
+    $tablePrefixes = '$databases[\'default\'][\'default\'][\'prefix\']= array(';
+    if ($config->userFramework === 'Backdrop') {
```

Comment:
I missed this during our clean-up yesterday. Since we know this will always be used for Drupal 7 projects, there's no need to check for Backdrop here